src/main/java/com/jfrog/ide/idea/configuration/ServerConfigImpl.java

```diff
@@ -417,6 +421,36 @@ public boolean readConnectionDetailsFromEnv() {
         return true;
     }
 
+    /**
+     * Read connection details using installed JFrog CLI.
```

Comment:
Read connection details using installed JFrog CLI.
-->
Read the connection details from JFrog CLI's config. The configuration is read by executing JFrog CLI.